jni/getroot.c

```diff
@@ -71,6 +71,9 @@ int modify_task_cred_uc(struct thread_info* __kernel info)
 		return 1;
 
 	tsp = malloc(sizeof(*tsp));
+	if (!tsp)
+		return 1;
```

Comment:
Thanks a ton, learning new things everyday :)

Pr is now updated, switched to -ENOMEM like you suggested and additionally in cmpsidstr() (was returning 1) which should be good too as of now.